UnityProject/Assets/Scripts/Editor/Tools/Networking.cs

```diff
@@ -82,9 +82,4 @@ private static void AdminPlzRestart()
 	{
 		GameManager.Instance.RestartRound();
 	}
-	[MenuItem("Networking/Extend round time")]
-	private static void ExtendRoundTime()
```

Comment:
Why would you remove this? Round time is still a thing

Reply:
This brings up an error whenever i bring it back.